helpers/test/controller.go

```diff
@@ -0,0 +1,65 @@
+package test_helpers
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+
+	"github.com/beego/beego/v2/server/web"
+	"github.com/onsi/ginkgo"
+)
+
+// HTTPRequest initiate new HTTP request and handle the error, will fail the test if there is any error
+func HTTPRequest(method, url string, body io.Reader) *http.Request {
```

Comment:
Regarding to our [convention](https://nimblehq.co/compass/development/code-conventions/golang/#functions). Prefer declaring the type for each parameter.

```suggestion
func HTTPRequest(method string, url string, body io.Reader) *http.Request {
```